src/Sylius/Bundle/AdminBundle/Doctrine/Query/Taxon/AllTaxons.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Doctrine\Query\Taxon;
+
+use Doctrine\Common\Collections\Criteria;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\ORM\EntityManagerInterface;
+use Sylius\Component\Locale\Context\LocaleContextInterface;
+use Sylius\Component\Locale\Context\LocaleNotFoundException;
+use Sylius\Component\Resource\Translation\Provider\TranslationLocaleProviderInterface;
+
+final class AllTaxons implements AllTaxonsInterface
+{
+    public function __construct(
+        private readonly EntityManagerInterface $entityManager,
+        private readonly LocaleContextInterface $localeContext,
+        private readonly TranslationLocaleProviderInterface $translationLocaleProvider,
+    ) {
+    }
+
+    public function getArrayResult(): array
+    {
+        $fallbackLocale = $this->translationLocaleProvider->getDefaultLocaleCode();
+        try {
+            $currentLocale = $this->localeContext->getLocaleCode();
+        } catch (LocaleNotFoundException) {
+            $currentLocale = $fallbackLocale;
+        }
+
+        $qb = $this->entityManager->getConnection()->createQueryBuilder();
```

Comment:
There is no need to shorten
```suggestion
        $queryBuilder = $this->entityManager->getConnection()->createQueryBuilder();
```